src/manager.ts

```diff
@@ -331,48 +331,48 @@ export class MenuManager {
     private static getScrollOffset() {
         let doc = this.curButton!.ownerDocument;
 
-        if (doc.body.style.top) {
-            return Math.abs(parseInt(doc.body.style.top, 10));
+        if (doc!.body.style.top) {
+            return Math.abs(parseInt(doc!.body.style.top, 10));
         }
 
-        if (doc.scrollingElement) {
-            return doc.scrollingElement.scrollTop;
+        if (doc!.scrollingElement) {
+            return doc!.scrollingElement.scrollTop;
         } else {
-            return doc.documentElement.scrollTop + doc.body.scrollTop;
+            return doc!.documentElement.scrollTop + doc!.body.scrollTop;
         }
     }
 
 
     private static blockScrolling(offset : number) {
         let doc = this.curButton!.ownerDocument;
```

Comment:
Are we able to do `let doc = this.curButton!.ownerDocument!` and then avoid adding `!` to doc everywhere below?

Reply:
updated!